src/assets/i18n/en_US.json

```diff
@@ -1220,6 +1222,10 @@
   "store_locator.results.single": "{{0}} store found:",
   "store_locator.title": "Store Finder",
   "subject.has.no.permission.assigned": "You do not have the permission to access this page. Please contact your account admin for access privileges.",
+  "subscriptions.not_subscribed": "not subscribed",
+  "subscriptions.recipient.already_subscribed.error": "Already subscribed to newsletter.",
```

Comment:
I have already subscribed to the newsletter.

Reply:
This error message is probably not needed anymore. Leave as is for now.